docs/property_handling.md

```diff
@@ -48,3 +69,14 @@ Properties can be returned to LM by setting facts with a prefix "output__". For
 The Ansible Lifecycle Driver will recognize this as a property to be returned to LM in the response. Any number of properties can be returned in this way.
 
 The prefix can be changed to something other than "output__" by setting the property "app.config.override.ansible.output_prop_prefix" when installing the driver using the Helm chart.
+
+Additionally, internal resource instances can be returned to LM by setting a fact named _associated_topology_.
+This is an example of a returned associated_topology in an ansible script:
+```
+- name set the associated topology
+  set_fact: 
+    associated_topology: 
+      stack_1: 
+        id: '8826f231-b7e2-424f-8764-2fdd6859940a'
+        type: 'Openstack'
+```
```

Comment:
Perhaps provide some context and indicate that this is returning associated topology for an Openstack stack, where the id is the stack id? Also, how would someone return multiple entries - what would that look like?